web/app/view/edit/Users.js

```diff
@@ -121,20 +121,24 @@ Ext.define('Traccar.view.edit.Users', {
         }, {
             text: Strings.userAdmin,
             dataIndex: 'admin',
+            renderer: Traccar.AttributeFormatter.getFormatter('admin'),
```

Comment:
Format boolean columns as translated "Yes/No" instead of just "true/false". Same in other places.